<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImgTestClass</class>
 <widget class="QMainWindow" name="ImgTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>遥感影像房屋提取系统 v1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="FileListDockWidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>111</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>波段列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="fileTreeView">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="inforDockWidget">
   <property name="windowTitle">
    <string>图像信息</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionImage"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView_Operation">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionZoom_In"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>提取</string>
    </property>
    <addaction name="actionExtract"/>
    <addaction name="actionFile_Save"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuView_Operation"/>
   <addaction name="menuWindow"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QDockWidget" name="extraction_result">
   <property name="windowTitle">
    <string>提取结果</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前状态：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QTextEdit" name="textEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionImage">
   <property name="text">
    <string>打开图像</string>
   </property>
   <property name="toolTip">
    <string>Open image files</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>文件信息</string>
   </property>
  </action>
  <action name="actionFile_Save">
   <property name="text">
    <string>保存提取结果</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭图像</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>放大</string>
   </property>
  </action>
  <action name="actionFit_Window">
   <property name="text">
    <string>Fit Window</string>
   </property>
  </action>
  <action name="actionNormal_Size">
   <property name="text">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionExtract">
   <property name="text">
    <string>提取</string>
   </property>
  </action>
  <action name="actionSaveresult">
   <property name="text">
    <string>保存提取结果</string>
   </property>
  </action>
  <zorder>extraction_result</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
